src/geo/map/infowindow-template.js

```diff
@@ -16,7 +16,8 @@ var InfowindowTemplate = Backbone.Model.extend({
     this.unset('fields');
   },
 
-  update: function (attrs) {
+  update: function (attrs, options) {
```

Comment:
(just curiosity)

why do we added options here?

Reply:
Not used anymore so removing this!